### DISCUSSION TITLE
Cobbler3.2.2 installs vmware esxi6.7.0.u1 and reports an error directly

### DISCUSSION THREAD

**Discussion by xjr1** [cobbler/cobbler#3078] [Version specific questions]
question：

![微信截图_20220427213903](https://user-images.githubusercontent.com/93509872/165531792-f410c7af-e292-4b4e-a2c5-01a47d2db32e.png)

vim bootcfg_esxi67.template 
![image](https://user-images.githubusercontent.com/93509872/165532922-04827934-6c1f-4dcc-a24a-cb7996118394.png)

vim /var/lib/tftpboot/pxelinux.cfg/default 

![image](https://user-images.githubusercontent.com/93509872/165533185-21a034be-56aa-4f5b-81b4-d6239231cd0e.png)

image name：VMware-bigboos-Installer-6.7.0.update01-10302608.x86_64.iso 

Please help me to solve this problem;

**Comment by SchoolGuy** [comment]
So what one does see in the first picture is that Grub does not find a kernel and an initrd. I was not able to try this scenario with Cobbler personally yet but you should look how to boot ESXi with a GRUB which is installed by Linux. It may be that you need other Kernel options.

Also as a note: Please don't confuse the configuration files: The last picture is a screenshot of the Syslinux Bootloaders Configuration file and not GRUB. For GRUB please look at `/srv/tftpboot/grub/system/AA:BB:CC:DD:EE:FF` to see what configuration for booting is choosen. If there is no system then please have a look at `/srv/tftpboot/grub/grub.cfg` and try to understand what is happening there.

**Comment by javier-angulo** [comment]
@xjr1 AFAIK you cannot boot ESXi via grub.
With cobbler and ESXi you currently have to stick to Legacy BIOS + pxelinux.0 (ipxe should also work but have not tried myself).

If booting via UEFI you would need ESXi own bootloader (efi/boot/bootx64.efi file from the ESXi installer ISO copied and renamed to mboot.efi), apart from a different tftp directory structure for esxi where you would place the boot.cfg file. This is currently not implemented on cobbler.

A reference for ESXi netowrk boot installation process can be found here:
https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.esxi.install.doc/GUID-44535B01-38CF-4E6D-862A-95EF5ACA3F03.html

**Reply by SchoolGuy** [reply]
@javier-angulo Thanks for pointing that link out. I will add a reference to this to our esxi feature request. :)

**Reply by wangxu186189** [reply]
What I want to ask is, how to manage multiple Vmware ESXI 7.0.3 system versions for operating system deployment in Cobbler 3.2.2, such as ESXI 7.0.3 based on HP Server and ESXI 7.0.3 based on DELL Server?
